From: Wang Ming <machel@vivo.com>
Subject: lib: remove error checking for debugfs_create_dir()
Date: Thu, 13 Jul 2023 16:24:43 +0800

It is expected that most callers should _ignore_ the errors return by
debugfs_create_dir() in err_inject_init().

Link: https://lkml.kernel.org/r/20230713082455.2415-1-machel@vivo.com
Signed-off-by: Wang Ming <machel@vivo.com>
Cc: Akinobu Mita <akinobu.mita@gmail.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/notifier-error-inject.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/lib/notifier-error-inject.c~lib-remove-error-checking-for-debugfs_create_dir
+++ a/lib/notifier-error-inject.c
@@ -83,9 +83,6 @@ static int __init err_inject_init(void)
 	notifier_err_inject_dir =
 		debugfs_create_dir("notifier-error-inject", NULL);
 
-	if (!notifier_err_inject_dir)
-		return -ENOMEM;
-
 	return 0;
 }
 
_
